services/app-api/src/postgres/contractAndRates/prismaSubmittedRateHelpers.ts

```diff
@@ -5,6 +5,18 @@ import {
     includeRateFormData,
 } from './prismaSharedContractRateHelpers'
 
+const includeFirstSubmittedRateRev = {
```

Comment:
nit: Should this be `includeLatestSubmittedRateRev`? From the first look I thought it was getting the first ever submitted revision, but looks like its looking at the most recent created revision that is submitted.

Reply:
I like it. more clear